x-pack/plugins/fleet/public/applications/fleet/sections/agents/agent_details_page/components/actions_menu.tsx

```diff
@@ -89,12 +101,26 @@ export const AgentDetailsActionMenu: React.FunctionComponent<{
       onClick={() => {
         setIsUpgradeModalOpen(true);
       }}
+      key="upgradeAgent"
     >
       <FormattedMessage
         id="xpack.fleet.agentList.upgradeOneButton"
         defaultMessage="Upgrade agent"
       />
     </EuiContextMenuItem>,
+    <EuiContextMenuItem
+      icon="inspect"
+      onClick={() => {
+        setIsContextMenuOpen(false);
+        setIsAgentDocumentFlyoutOpen(!isAgentDocumentFlyoutOpen);
+      }}
+      key="agentDocument"
+    >
+      <FormattedMessage
+        id="xpack.fleet.agentList.viewAgentDocumentText"
+        defaultMessage="View agent document"
```

Comment:
We try to be as descriptive with labeling as possible so that users have a better idea of what an option will do for them.

Could we use "View agent json" instead of `"View agent document"`?  My first thought was "View agent details,", but that label is already being used.

Reply:
Thank you @karenzone - I am not sure whether there is a UX pattern in this case. `View agent json` sounds clear to me. There is a sort of equivalent in the agent policy simply labelled `View policy` (note that the output is represented as yaml, not json, like in the screenshot @jen-huang posted above), so maybe `View agent` could even make sense? Or maybe `View full agent details`?

@zombieFox As it looks like Karen is on PTO this week could you weigh in on this?

cc @jen-huang @kpollich 

<img width="1233" alt="Screenshot 2023-04-17 at 15 26 20" src="https://user-images.githubusercontent.com/23701614/232498481-192ae62c-eabd-4619-8a5e-71795388b269.png">